### ISSUE TITLE
`badge` affects size of the icons, even if `size` is set, but it doesn't affect anything, when `size` is not set

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2741]
<img width="756" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/92883a05-bf86-4384-b063-085ab7128a72">

1. Badge should just add space, no matter what size
2. Size should always result in the same visual size

**Comment by adrianschmidt** [comment]
It looks to me like the icons are the same size with and without `badge` in your screenshot, so I'm not sure what this issue is about?

When you say the "the icon should be the same size", do you mean that "the whole component should have the same outer size, so to add the extra padding, the icon should be made smaller"? That's the only way I can figure out to interpret this issue 😅

**Comment by Kiarokh** [comment]
Yes. I refer to the whole element/component as the "icon". In UI design, this is a problematic and unpredictable issue. Since you want the accumulated visual size of the icon-element to be a certain size, and it shouldn't matter if there is a badge (or colored backdrop layer) behind it or not.

**Comment by adrianschmidt** [comment]
Alright, then I understand.

Also: 🙋‍♂️ _guilty!_

😂

Well, I'm not sure if I'm guilty of the "but only when the `size` attribute is also set"-part (although I probably am), but I'm _definitely_ guilty of making the decision that the padding adds to the total size of the component 🙈 😅

Feel absolutely free to fix that! 🤗
